src/components/03-components/01-actions/actions.config.yml

```diff
@@ -1,17 +1,5 @@
 title: Actions Menu
 status: wip
 preview: "@sam-content"
-context:
```

Comment:
Update preview to "@sam" or "@sam-grid" to avoid unintended consequences of `usa-prose`